client/src/components/profileInfoEdit/ProfileDeleteModal.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Delete Alert Modal component display a modal and ask for confirmation when the delete button is clicked
+ */
+import React from 'react';
+ import toast from 'react-hot-toast';
+ import { Button, Modal } from 'react-bootstrap'; 
+ import { useSelector, useDispatch } from 'react-redux';
+ import { logout } from '../../actions/auth';
+ import { deleteTeam ,deleteGHPages} from '../../actions/team';
+
+
+  const ProfileDeleteModal = ({ deleteAlert, setdeleteAlert }) => {
+   
+   const {teamId} = useSelector((state) => state.team,);
+   const dispatch = useDispatch();
+
+   const fullDelete = () => {
+    
+    const access_token = localStorage.getItem('GH_access_token');
+    if(access_token===null) {
+      toast.error('Log in with github account')
+    } 
+    else{
+      try{
+      dispatch(deleteGHPages(teamId,access_token))
+      dispatch(deleteTeam(teamId))
+      dispatch(logout())
+      } catch(error){
+        toast.error('GitHub Pages doesnt exist')
+      }
+      }   
+   };
+   const partialDelete = () => {
+    dispatch(deleteTeam(teamId))
+    dispatch(logout())
+    toast.success(`profile has been successfully deleted`);
+   };
+   return (
+     <Modal show={deleteAlert}>
+       <Modal.Header className="modalHeader">
+         <Modal.Title> Delete Account! </Modal.Title>
+       </Modal.Header>
+       <Modal.Body>
+         Are you sure you want to delete your account? All significant data will be deleted!
+       </Modal.Body>
+       <Modal.Footer>
+       <Button variant="light" onClick={() => setdeleteAlert(false)}>
+           back
+         </Button>
+         <Button variant="danger" onClick={partialDelete}>
+           Keep github page
```

Comment:
Same here with capitalization "gihub page"

Reply:
Either the buttons layout or color should change. Hard to distinguish delete all data and keep github page button. (For accessibility)

Reply:
Also my 2c, but I think the two buttons should be on the profile page rather than in this modal, this modal should just be  for confirming whether you want to delete your account. 
Deleting the account means everything needs to go, the data in the DB + repo. 
Deleting the data should mean the data in the DB (except for team doc since that's the account) + repo as well imo because would you want to keep the data in the repo if you're starting from a clean state?
It doesn't make sense to keep the GH pages site if you're leaving the site, or if you want to start the site again.
Anybody, feel free to add your own thoughts to the discussion.

Reply:
So do you reckon have Clear all data and Delete account as the two buttons instead?

Reply:
Imo we thought we'd have two options,
delete all data + repo and delete website data but keep the repo (not for restarting )
but yeah this can be easily refactored to the functionalities above!